js/build_map.js

```diff
@@ -98,17 +98,17 @@ if(waterUseViz.interactionMode === 'tap') {
   stateDataFile = "data/state_boundaries_USA.json";
 }
 
-d3.json(stateDataFile, function(error, data1) {
+d3.json(stateDataFile, function(error, stateBoundsRaw) {
 	if (error) throw error;
-  d3.tsv("data/county_centroids_wu.tsv", function(error, data2) {
+  d3.tsv("data/county_centroids_wu.tsv", function(error, countyCentroids) {
 	  if (error) throw error;
-    d3.json("data/wu_data_15_range.json", function(error, data3) {
+    d3.json("data/wu_data_15_range.json", function(error, waterUseRange) {
 	    if (error) throw error;
-      d3.json("data/wu_data_15_sum.json", function(error, data4) {
+      d3.json("data/wu_data_15_sum.json", function(error, waterUseNational) {
 	      if (error) throw error;
```

Comment:
OH but are you also suggesting creating globals for the ones we don't put into waterUseViz already? That's where I am a little concerned. I don't want to push stateBoundsUSA into the global thing because it might get complicated

Reply:
The way I handle the state bounds and county centroids will change slightly when I work on getting fillMap split into a fxn for making a map and a separate fxn for adding the circles. That will come in a different PR.

Reply:
Didn't notice that there were some that were never global - agreed that they should just get passed to `fillMap` as before. I like the change you just made, assigning immediately to globals if and only if they need to be globals.